packages/progress/src/progress.vue

```diff
@@ -33,7 +33,12 @@
       :style="{fontSize: progressTextSize + 'px'}"
     >
       <template v-if="!status">{{percentage}}%</template>
-      <i v-else :class="iconClass"></i>
+      <template v-if="status && status === 'text'">
```

Comment:
Simplify your code.

```suggestion
      <template e-lse>
        <slot v-if="status === 'text'"></slot>
        <i v-else :class="iconClass"></i>
      </template>
```

Reply:
Thanks, that's better. but what is e-lse? did you mean v-else? or that's a new API!!!